CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## v1.21.0 (28/10/2024)
+
+**Common**
+* Removed deprecated `olp::client::OlpClient::SetSettings(...)` method.  Use `olp::client::OlpClient` constructor instead.
+* Removed deprecated `olp::client::HttpResponse::status` field. Use `HttpResponse::GetStatus()` method instead.
+* Removed deprecated `olp::client::HttpResponse::response` field. Use `HttpResponse::GetRawResponse()` method instead.
+* Removed deprecated `olp::client::HttpResponse::headers` field. Use `HttpResponse::GetHeaders()` method instead.
+
+**olp-cpp-sdk-core**
+* Added `olp::client::HttpResponse::GetResponseAsBytes` convenience method to get the response body as a vector of unsigned chars.
+* Added `olp::client::HttpResponse::GetResponseAsString` convenience method to get the response body content as a string.
+* Added `olp::client::HttpResponse::GetRawResponse` convenience method to get reference to the response object.
+* Added `olp::client::OlpClient::GetSettings()` method.
+* Removed deprecated `olp::http::CreateDefaultNetwork(size_t)` function. Use `olp::http::CreateDefaultNetwork(NetworkInitializationSettings)` instead.
+* Removed deprecated `olp::http::NetworkSettings::GetRetries()` method.
+* Removed deprecated `olp::http::NetworkSettings::WithRetries()` method.
+* Removed deprecated `olp::http::NetworkSettings::GetConnectionTimeout()` method. Use `NetworkSettings::GetConnectionTimeoutDuration()` instead.
+* Removed deprecated `olp::http::NetworkSettings::WithConnectionTimeout(int)` method. Use `NetworkSettings::WithConnectionTimeout(std::chrono::milliseconds)` instead.
+* Removed deprecated `olp::http::NetworkSettings::GetTransferTimeout()` method. Use `NetworkSettings::GetTransferTimeoutDuration()` instead.
+* Removed deprecated `olp::http::NetworkSettings::WithTransferTimeout(int)` method. Use `NetworkSettings::WithTransferTimeout(std::chrono::milliseconds)` instead.
+* Removed deprecated `olp::utils::Dir::exists(const std::string&)` method. Use `Dir::Exists(...)` instead.
+* Removed deprecated `olp::utils::Dir::remove(const std::string&)` method. Use `Dir::Remove(...)` instead.
+* Removed deprecated `olp::utils::Dir::create(const std::string&)` method. Use `Dir::Create(...)` instead.
+* Changed cURL network implementation requirements. OpenSSL is mandatory now.
+* Fixed asynchronous access to list of cancelled tasks to skip callback from in iOS downloading tasks restart for background mode.
```

Comment:
This sentence is difficult to decipher. When it gets to "callback from in iOS" I can't really understand it. Can you explain what you wanted to say or try to edit this sentence for clarity? It might be better to try to convey this information in two shorter sentences rather than a long one, too.